zkevm-circuits/src/mpt_circuit/helpers.rs

```diff
@@ -1170,7 +1157,7 @@ impl<F: Field> WrongGadget<F> {
         let mut config = WrongGadget::default();
         circuit!([meta, cb.base], {
             // Get the previous key data
-            ifx! {is_non_existing, not!(is_in_empty_tree) => {
+            ifx! {(is_non_existing, not!(is_in_empty_tree)) => {
```

Comment:
Extra parentheses necessary in ifs with multiple inputs?

Reply:
Not really, cuz this is the only place where we have two conditions in `ifx!`, with the new macro I needed this paran for `$condition:tt`.